wg-api/spec-documents/login-item-api.md

```diff
@@ -0,0 +1,80 @@
+# `app.{set|get}LoginItemSettings(settings)`
+
+## Summary
+
+`app.setLoginItemSettings(settings)` and `app.getLoginItemSettings(settings)` will be updated on macOS 13 to use a single streamlined underlying API on both Mac App Store and regular macOS builds. This will remove several configuration and settings options, including `openAsHidden` in `app.setLoginItemSettings` and `wasOpenedAtLogin` and `wasOpenedAsHidden` in `app.getLoginItemSettings`.
+
+This will simplify the underlying logic in accordance to the APIs apple has outlined for usage and improve parity between MAS and non-MAS builds. This will also allow developers to register app service objects, launch daemons, and login item services to launch on startup.
+
+## Platforms
+
+**macOS**
+
+## Impetus
+
+At present, Electron uses `LSSharedFileList` APIs to register applications as login items. These APIs have been deprecated since 10.11, and have not worked for several years on sandboxed Mac App Store builds. Chromium is also slowly [removing usage upstream](https://chromium-review.googlesource.com/c/chromium/src/+/3997795), requiring us to float increasingly deprecated code. As of Ventura, several key aspects of login item settings, 
+namely `openAsHidden`, also no longer work.
+
+In macOS 13, Apple introduced [`SMAppService`](https://developer.apple.com/documentation/servicemanagement/smappservice?language=objc) to take the place of the APIs they phased out. This service is used to register and control `LoginItems`, `LaunchAgents`, and `LaunchDaemons` as helper executables for apps.
+
+## API Design
+
+The new methods will adhere to the following semantics:
```

Comment:
It looks like these have the same name & a (nearly) compatible type signature with the existing methods, is that right?

Reply:
Yes - just property differences!